### DISCUSSION TITLE
Flycast Hotkeys

### DISCUSSION THREAD

**Discussion by Velvetbruh** [flyinghead/flycast#698] [Q&A]
I recently started using Flycast but i cant figure out any hotkeys or if there are any at all. I found alt+enter to get full screen but cant find any others id appreciate if someone could tell me or maybe give a list of all the hotkeys you can use while in game and not running anything.
Edit: nvm i figured it out a minute after asking this but cant delete it so yeah

**Comment by flyinghead** [comment]
Fixed hotkeys:
Toggle Fullscreen: ALT+Enter
Toggle mouse capture: Left ALT+Left CTRL

Configurable hotkeys:
Open Menu/Pause: TAB (by default)
Toggle Fast forward: Space (by default)

**Reply by Blackgrowl** [reply]
> Fixed hotkeys: Toggle Fullscreen: ALT+Enter Toggle mouse capture: Left ALT+Left CTRL
> 
> Configurable hotkeys: Open Menu/Pause: TAB (by default) Toggle Fast forward: Space (by default)

Are there any hotkeys for Pause? Specifically, System Pause, just like in every other emulator like FinalBurn Neo, Dolphin and Duckstation. If not, where may I suggest to add that for the next update?

**Reply by flyinghead** [reply]
Open Menu/**Pause**: TAB (by default)

**Reply by Blackgrowl** [reply]
Well, that works, but it also opens the menu which blocks the way, I'd prefer it if it simply paused so I can take a screenshot or read the subtitles, especially on an arcade game.
